tools/src/main/java/org/bitcoinj/tools/WalletTool.java

```diff
@@ -1384,6 +1385,14 @@ private static void createWallet(OptionSet options, NetworkParameters params, Fi
         } else {
             wallet = Wallet.createDeterministic(params, outputScriptType);
         }
+        // add BIP44 key chain
+        // BIP44 Chain
+        DeterministicKeyChain bip44Chain = DeterministicKeyChain.builder()
+                .seed(wallet.getKeyChainSeed())
+                .accountPath(DerivationPathFactory.get(params).bip44DerivationPath(0))
+                .build();
+        wallet.addAndActivateHDChain(bip44Chain);
+
```

Comment:
This will allow WalletTool to use the same standard as the Dash Wallet app to create the wallets.  

We can take a Dash Wallet recovery phrase and restore a wallet with it to use with various DashJ tools and examples which was helpful with investigating sync issues.